### PR TITLE
Optimize SVG

### PR DESCRIPTION
**Pull request by igoradamenko**
### What did you implement:

I've optimized SVGs that are used for icons.

### How did you implement it:

By removing useless attributes & tags and using SVGOMG in some cases.

### How can we verify it:

By opening the HTML file stored in this ZIP:

[index.zip](https://github.com/essentialkaos/sonar/files/7610836/index.zip)

The HTML file contains original & edited SVGs, so you can compare them one by one in any browser. I've changed only the IDs of the original SVGs to make everything work.

### TODO's:

- [x] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

**Is this ready for review?:** Yes
**Is it a breaking change?:** No

> Write documentation

Not applicable.

> Provide verification config / commands

Not sure what it means 🤔

### CODE REVIEW

File: svg/svg.go [hunk 1]
```diff
@@ -11,28 +11,28 @@ package svg
 func GetBullet(color string) string {
 	switch color {
 	case "":
-		return `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="none" width="16" height="16"><defs><path fill="#fff" fill-opacity="0" d="M16 0H0v16h16V0z" id="a"/></defs><use xlink:href="#a"/></svg>`
```
Comment (igoradamenko) [inline svg/svg.go:14]: `preserveAspectRatio="none"` this thing is useless if an SVG does not have `viewBox` attribute:

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio

File: svg/svg.go [hunk 2]
```diff
@@ -11,28 +11,28 @@ package svg
...
+		return `<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"></svg>`
 	default:
-		return `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="none" width="16" height="16"><defs><path fill="` + color + `" d="M5.85 13.1q.9.9 2.15.9 1.3 0 2.1-.9.9-.85.9-2.1t-.9-2.15Q9.3 8 8 8q-1.25 0-2.15.85Q5 9.75 5 11t.85 2.1z" id="a"/></defs><use xlink:href="#a"/></svg>`
```
Comment (igoradamenko) [inline svg/svg.go:16]: Most of the changes are just unwrapping the `defs` node & removing the ID of the path nested in it.

File: svg/svg.go [hunk 3]
```diff
@@ -11,28 +11,28 @@ package svg
...
+	return `<svg xmlns="http://www.w3.org/2000/svg" width="20" height="16"><path fill="#589ABF" d="M18.35 2.5q-.4-.4-1.1-.5-.65-.05-1.4.3-.75.35-1.35.95l-1.7 1.7L8 3.55h-.5q-.2.1-.4.3l-1 1q-.3.3-.3.8.1.4.45.75L9.3 8.45l-1 1-1.15-.25q-.25-.05-.45.05-.3 0-.5.2l-.9.9q-.3.3-.3.8.05.45.5.7l2.1 1.4 1.4 2.1q.25.45.7.5.5 0 .8-.3l.9-.9q.2-.2.25-.45.05-.25 0-.5l-.25-1.15 1-1 2.05 3.05q.35.35.75.45.5 0 .8-.3l1-1q.2-.2.3-.4v-.5l-1.4-4.8 1.7-1.7q.6-.6.95-1.35.35-.75.3-1.4-.1-.7-.5-1.1m-2.9 1.7q.55-.55 1.15-.75.6-.3.85-.05t-.05.9q-.15.6-.7 1.15l-2.3 2.3L15.95 13l-.55.55-2.75-4.05L10 12.15l.35 1.65-.5.5-1.3-2-2-1.3.5-.5 1.7.4 2.65-2.65-4.1-2.8.55-.55 5.3 1.6 2.3-2.3z"/></svg>`
```
Comment (igoradamenko) [inline svg/svg.go:32]: For this SVG and the one below I've also removed a `transform` attribute using SVGOMG, which is shifted the whole shape in the direction of the `transform`.

**Comment by andyone** [comment]
Thanks for your contribution!

> Not sure what it means 🤔

It's usually required for testing new features.

**Comment by igoradamenko** [comment]
Ah, okay, got it 🐨

**Comment by andyone** [comment]
Looks good, merged. Thanks!

**Pull request merged**